crates/compiler/test_gen/src/helpers/llvm.rs

```diff
@@ -540,6 +540,7 @@ pub fn try_run_lib_function<T>(
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Is the plan to use this code later, so we are avoiding removing it even if it's dead? This seems like a recipe for dead code falling through the cracks, though I'm sure it would get cleaned up _eventually_.

Reply:
Some of these test helpers are considered dead code when they get imported by tests in a `tests/` directory (which are treated as their own crates) but the test doesn't use all of them. So they aren't dead in the sense that if you take them out, some tests (but not others) will no longer compile, but depending on the test that's running, they can generate annoying warnings about dead code.

There's probably a better way to do this, but until we know what it is, this is the quick fix. 😄

Reply:
#[cfg(test)]?